### PR TITLE
picking up latest P1 dependencies

### PR DESCRIPTION
**Pull request by dlockhart**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -5,13 +5,13 @@
       "d2l-accordion": "0.1.1",
       "d2l-activity-alignments": "0.0.27",
       "d2l-alert": "3.2.3",
-      "d2l-button": "4.10.0",
+      "d2l-button": "4.10.1",
```
Comment (dlockhart) [inline bower.json]: @dbatiste FYI

File: bower.json [hunk 2]
```diff
@@ -5,13 +5,13 @@
...
       "d2l-button-group": "2.1.2",
       "d2l-card": "4.1.20",
       "d2l-colors": "3.1.3",
       "d2l-course-image": "2.1.7",
       "d2l-dnd-sortable": "0.0.4",
-      "d2l-dropdown": "6.6.5",
+      "d2l-dropdown": "6.6.6",
```
Comment (dlockhart) [inline bower.json]: @dbatiste FYI

File: bower.json [hunk 3]
```diff
@@ -22,7 +22,7 @@
       "d2l-hierarchical-view": "1.0.5",
       "d2l-hm-constants-behavior": "5.31.0",
       "d2l-html-editor": "0.2.20",
-      "d2l-icons": "5.1.5",
+      "d2l-icons": "5.1.6",
```
Comment (dlockhart) [inline bower.json]: @dbatiste FYI

File: bower.json [hunk 4]
```diff
@@ -35,7 +35,7 @@
       "d2l-more-less": "4.3.1",
       "d2l-my-courses": "6.0.14",
       "d2l-my-dashboards": "1.1.0",
-      "d2l-navigation": "2.2.3",
+      "d2l-navigation": "2.3.0",
```
Comment (dlockhart) [inline bower.json]: @ctwomey1 this is getting pulled into the Polymer 1 branch of BSI for 10.8.11 -- let me know if that's not OK

File: bower.json [hunk 5]
```diff
@@ -45,13 +45,13 @@
       "d2l-polymer-siren-behaviors": "0.3.1",
       "d2l-resize-observer-polyfill-import": "1.5.1",
       "d2l-rubric": "0.156.0",
-      "d2l-save-status": "1.6.0",
+      "d2l-save-status": "1.7.0",
```
Comment (dlockhart) [inline bower.json]: @dabdullahd2l FYI

File: bower.json [hunk 6]
```diff
@@ -45,13 +45,13 @@
...
       "d2l-scroll-spy": "0.0.3",
       "d2l-search-widget": "3.1.0",
       "d2l-simple-overlay": "2.3.1",
       "d2l-status-indicator": "2.0.0",
       "d2l-table": "1.7.6",
-      "d2l-tabs": "0.4.5",
+      "d2l-tabs": "0.4.6",
```
Comment (dlockhart) [inline bower.json]: @dbatiste FYI

File: bower.json (hunk 4 above)
Comment (ctwomey1) [reply bower.json]: Shouldn't be a problem, but not required

**Pull request merged**